xbmc/video/windows/GUIWindowVideoNav.cpp

```diff
@@ -491,14 +492,24 @@ bool CGUIWindowVideoNav::GetDirectory(const std::string &strDirectory, CFileItem
     }
 
     CVideoDbUrl videoUrl;
-    if (videoUrl.FromString(items.GetPath()) && items.GetContent() == "tags" &&
-       !items.Contains("newtag://" + videoUrl.GetType()))
+    if (videoUrl.FromString(items.GetPath()))
     {
-      CFileItemPtr newTag(new CFileItem("newtag://" + videoUrl.GetType(), false));
-      newTag->SetLabel(g_localizeStrings.Get(20462));
-      newTag->SetLabelPreformatted(true);
-      newTag->SetSpecialSort(SortSpecialOnTop);
-      items.Add(newTag);
+      if (items.GetContent() == "tags" && !items.Contains("newtag://" + videoUrl.GetType()))
+      {
+        CFileItemPtr newTag(new CFileItem("newtag://" + videoUrl.GetType(), false));
```

Comment:
`const auto newTag{std::make_shared<CFileItem>(...`

Reply:
We want to get rid of `CFileItemPtr`, use the real type instead, please.

Reply:
I'll replace them. Too much code, it's hard to fix the critical issues and applying code format at the same time. I fixed some, and these seem missed.